Refs #37601 - Refresh/deploy CA cert on hosts (#3193)
theforeman/foreman-documentation: guides/common/modules/proc_planning-for-ca-certificate-renewal.adoc

Author: I'm not sure I'm using correct terminology. Please check.

[id="planning-for-ca-certificate-renewal"]
= Planning for CA certificate renewal

> If you need to update SSL certificates in the Certification Authority (CA) certificate on your {ProjectServer}, use a temporary dual CA certificate to retain the HTTPS connections to your {ProjectServer} during the renewal.

Reviewer: ```suggestion
If you need to update Certification Authority (CA) certificate on your {ProjectServer}, add the new CA certificate and use a temporary dual CA certificate file to retain the HTTPS connections to your {ProjectServer} during the renewal.
```

> If you need to update Certification Authority (CA) certificate on your {ProjectServer}, add the new CA certificate and use a temporary dual CA certificate file to retain the HTTPS connections to your {ProjectServer} during the renewal.

Reviewer 2: ```suggestion
If you need to update the Certification Authority (CA) certificate on your {ProjectServer}, add the new CA certificate and use a temporary dual CA certificate file to retain the HTTPS connections to your {ProjectServer} during the renewal.
```

.Procedure

> . Add the new SSL certificate to the CA certificate of {ProjectServer} and keep the old SSL certificate in the CA.

Reviewer: ```suggestion
. Add the new SSL certificate to the CA certificate file on {ProjectServer} and keep the old SSL certificate.
```

> . Renew the certificates on {ProjectServer} and any {SmartProxyServer}s.

Reviewer 3: ```suggestion
. Renew the certificates on {ProjectServer} and any {SmartProxyServers}.
```

see https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L145

. Deploy the dual CA certificate on hosts.

> . Replace the dual CA certificate on {ProjectServer} with a new CA certificate that contains only the new SSL certificate.

Reviewer: ```suggestion
. Remove the old certificate from the CA certificates file on {ProjectServer}, so the CA certificate file contains only the new SSL certificate.
```

> . Renew the certificates on {ProjectServer} and any {SmartProxyServer}s.

Reviewer 3: ```suggestion
. Renew the certificates on {ProjectServer} and any {SmartProxyServers}.
```

as above.
